spec/integration/util/windows/security_spec.rb

```diff
@@ -407,14 +418,18 @@ def grant_everyone_full_access(path)
         end
 
         describe "#group=" do
+          it "should accept the test group" do
+            check_group_depending_on_current_user(path)
```

Comment:
Actually, if the user is :system then this doesn't help, I'll make this set group to :guest to ensure there's a positive test.